### ISSUE TITLE
Can I move the display of opponents to another location?

### ISSUE THREAD

**Issue by AnOpenHeresy** [tapahob/BG2RadarOverlay#25]
By, say, 50/20 pixels?

**Comment by tapahob** [comment]
Its better to make it draggable, I think.

**Comment by myevilmode** [comment]
Yes, but simple coordinate x y in ini file will be fine for me.

**Comment by tapahob** [comment]
> Yes, but simple coordinate x y in ini file will be fine for me.

I moved it a bit in the latest release and added the X offset for you in the config file

**Comment by myevilmode** [comment]
Thanks a lot but how to configure it? should i enter x20y30? Or how?

**Comment by tapahob** [comment]
> Thanks a lot but how to configure it? should i enter x20y30? Or how?

It is an X axis offset - by default it is 0 and you can set it to any integer value like 30 or -20

**Comment by myevilmode** [comment]
There is a black pixel in the upper left corner of the screen. I can move it, can I save the position of this black pixel?
![BG_Radar_Overlay_oJdiupfn9A](https://user-images.githubusercontent.com/51825875/190921376-44e49cab-19d1-4149-8652-9675e90dfc1e.jpg)

**Comment by tapahob** [comment]
> There is a black pixel in the upper left corner of the screen. I can move it, can I save the position of this black pixel?
Oh I didn't know that. Currently you can't save it but what you can do is to move it to wherever you want, take a screenshot and open it in paint and measure the X distance from the left. Current default position for it is 130 so you subtract 130 from the value you've measured and put it into the X offset in config.

**Comment by myevilmode** [comment]
Ok, thanks actually it's nothing difficult to change the position every time I start the game and that's what I meant :)

**Comment by tapahob** [comment]
> Ok, thanks actually it's nothing difficult to change the position every time I start the game and that's what I meant :)

Gotta make a proper positioning functionality once I have time for that

**Comment by myevilmode** [comment]
Great idea, good luck!

**Comment by tapahob** [comment]
For the record, I actually fixed the enemy list offset in the latest commit. The offset param from the config should work now.
I didn't make a release with the fix as it is a very minor, mb once I do the drag I'll compile one.

**Comment by tapahob** [comment]
> Hello! Did you made that ability to change position of the bar?
Yes. In the config.cfg file there is a `EnemyListXOffset` parameter now. It is set to zero by default but you can put there the number of pixels you want it to be offset by.

**Comment by tapahob** [comment]
Strictly speaking, the parameter there was already there before, but its only in the latest release when i got it to work properly
